integration/mocha-tests/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
can we reuse the same ts config from the main library https://github.com/aws/amazon-chime-sdk-js/tree/main/config

Reply:
Actually there is no ts file then why do we need it ?

Reply:
Actually there is no ts file then why do we need it ?